apps/src/javalab/JavalabEditor.jsx

```diff
@@ -579,21 +577,26 @@ class JavalabEditor extends React.Component {
                 );
               })}
             </Nav>
-            <Tab.Content animation={false}>
+            <Tab.Content id="tab-content" animation={false}>
               {orderedTabKeys.map(tabKey => {
                 return (
                   <Tab.Pane eventKey={tabKey} key={`${tabKey}-content`}>
                     <div
                       ref={el => (this._codeMirrors[tabKey] = el)}
                       style={{
-                        ...style.editor,
-                        ...(isDarkMode && style.darkBackground)
+                        ...styles.editor,
+                        ...(isDarkMode && styles.darkBackground)
```

Comment:
(continued from note above)
you can add in here
`...{height: this.props.height}` <- (syntax may be a little off. )

Reply:
Done!